[BUG] 'VisualStudioCodeCredential' is obsolete causing net-core-ci failure during mgmt PR

### Library name and version

Azure.Developer.MicrosoftPlaywrightTesting.TestLogger

### Describe the bug

```
sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/PlaywrightServiceOptions.cs(94,24): error CS0618: 'VisualStudioCodeCredential' is obsolete: 'This credential is deprecated because the VS Code Azure Account extension on which this credential relies has been deprecated. Consider using other dev-time credentials, such as VisualStudioCredential, AzureCliCredential, AzureDeveloperCliCredential, AzurePowerShellCredential. See the Azure Account extension deprecation notice here: https://github.com/microsoft/vscode-azure-account/issues/964.'
```
net-core-ci is failing with above error [here](https://dev.azure.com/azure-sdk/public/_build/results?buildId=4704455&view=logs&j=af736694-cf58-5d76-4d00-61d125b935ef&t=4dfa7e99-2da8-50a1-7d55-3cd4dbc94004&l=117)

### Expected behavior

net-core-ci should pass

### Actual behavior

net-core-ci is failing

### Reproduction Steps

https://github.com/Azure/azure-sdk-for-net/pull/49110/checks?check_run_id=39675717835

### Environment

_No response_

--- a/sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/Constants.cs
+++ b/sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/Constants.cs
@@ -100,11 +100,6 @@ internal class AzureTokenCredentialType
     public static readonly string VisualStudioCredential = "VisualStudioCredential";
 
     /// <summary>
-    /// Visual Studio Code Credential.
-    /// </summary>
-    public static readonly string VisualStudioCodeCredential = "VisualStudioCodeCredential";
-
-    /// <summary>
     /// Azure CLI Credential.
     /// </summary>
     public static readonly string AzureCliCredential = "AzureCliCredential";

--- a/sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/PlaywrightServiceOptions.cs
+++ b/sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/PlaywrightServiceOptions.cs
@@ -89,10 +89,6 @@ public class PlaywrightServiceOptions
         {
             return new VisualStudioCredential();
         }
-        else if (azureTokenCredentialType == AzureTokenCredentialType.VisualStudioCodeCredential)
-        {
-            return new VisualStudioCodeCredential();
-        }
         else if (azureTokenCredentialType == AzureTokenCredentialType.InteractiveBrowserCredential)
         {
             return new InteractiveBrowserCredential();

--- a/sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/tests/PlaywrightServiceOptionsTest.cs
+++ b/sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/tests/PlaywrightServiceOptionsTest.cs
@@ -75,7 +75,6 @@ public class PlaywrightServiceOptionsTest
     [TestCase("InteractiveBrowserCredential", typeof(InteractiveBrowserCredential))]
     [TestCase("SharedTokenCacheCredential", typeof(SharedTokenCacheCredential))]
     [TestCase("VisualStudioCredential", typeof(VisualStudioCredential))]
-    [TestCase("VisualStudioCodeCredential", typeof(VisualStudioCodeCredential))]
     [TestCase("DefaultAzureCredential", typeof(DefaultAzureCredential))]
     [TestCase("", typeof(DefaultAzureCredential))]
     [TestCase(null, typeof(DefaultAzureCredential))]

